### DISCUSSION TITLE
Will you distribute bins on GitHub going forward?

### DISCUSSION THREAD

**Discussion by brian6932** [erengy/taiga#1091] [Questions] [answered]
This is not really a bug report or a feature request, but more of a personal issue. I made a package that updates using the bins distrubuted on GitHub (which includes pre-releases), and I wanted to know whether or not going forward bins would be hosted on `/releases` as well?
I only ask this because https://taiga.moe/ hosted pre-releases with a query param instead of on the release itself, and also because [v1.4.0 did not contain any bins on gh](https://github.com/erengy/taiga/releases/tag/v1.4.0 ).

**Comment by erengy** [comment, marked as answer]
Yes, I'm planning to make better use of releases and possibly provide automatic pre-releases via GitHub Actions (#1024).

I updated v1.4.0 and other previous releases missing a binary now.
